config/samples/glance_v1beta1_glanceapi.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
 spec:
   serviceUser: glance
   serviceAccount: glance
-  containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
```

Comment:
@dprince we can use this patch as a follow up of PR #358 and remove this line here using and not fail because of the `glanceAPI` webhook.